clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/ingestion/heartbeat/HeartbeatMonitoringService.java

```diff
@@ -339,6 +340,47 @@ public void updateLagMonitor(
     }
   }
 
+  /**
+   * Get maximum heartbeat lag from all regions for a given LEADER replica.
+   * @return Max leader heartbeat lag, or Long.MAX_VALUE if any region's heartbeat is unknown.
+   */
+  public long getReplicaLeaderHeartbeatLag(String storeName, int version, int partitionId) {
```

Comment:
Let's somehow add the notion that this is region impacting in the method name?  Or that it's the MOST lagged region?  Basically, when intellisense kicks in, what makes sure I'm aware this is happening.